pinc/stages.inc

```diff
@@ -411,7 +411,7 @@ function get_reserve_length($project, $round)
         }
 
         // Otherwise, reserve for 7 days.
```

Comment:
We should either update this comment to match, or change the comment to not have a number, something like `Otherwise, reserve for a few days`

Reply:
Oops, missed that. How about a generic comment like
```
Otherwise, update as requested by GM
```

Reply:
and done.